lib/view_model/send/output.dart

```diff
@@ -152,7 +152,10 @@ abstract class OutputBase with Store {
   }
 
   WalletType get walletType => _wallet.type;
-  final CryptoCurrency Function() cryptoCurrencyHandler;
+
+  @observable
+  CryptoCurrency Function() cryptoCurrencyHandler;
```

Comment:
this can stay final and you can use `selectedCryptoCurrency` variable always since it's a function that would be called when needed and the value would be updated